meta-cube/recipes-core/lxc/files/lxc-overlayclean

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -
+
+# /usr/lib64/lxc/lxc/lxc-overlayclean
+# Final cleanup for lxc overlay related services
+
+cp /etc/lxc/lxc-overlayrestore.bak /etc/lxc/lxc-overlayrestore
```

Comment:
it is unclear, why we have a .bak file being copied into lxc-overlayrestore. There should be a better comment here.